Index: src/test/java/com/appdetex/restserver/RestServerApplicationTests.java
===================================================================
diff --git a/src/test/java/com/appdetex/restserver/RestServerApplicationTests.java b/src/test/java/com/appdetex/restserver/RestServerApplicationTests.java
deleted file mode 100644
--- a/src/test/java/com/appdetex/restserver/RestServerApplicationTests.java	(revision 29d98c7b934bd4cc69b77ee85b4b595717524bc4)
+++ /dev/null	(revision 29d98c7b934bd4cc69b77ee85b4b595717524bc4)
@@ -1,18 +0,0 @@
-package com.appdetex.restserver;
-
-import org.junit.jupiter.api.Test;
-import org.springframework.boot.test.context.SpringBootTest;
-
-@SpringBootTest
-class RestServerApplicationTests {
-
-	@Test
-	void contextLoads() {
-	}
-
-	@Test
-	void TestRuleJacuzziBrand(){
-
-	}
-
-}
Index: .idea/compiler.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<project version=\"4\">\n  <component name=\"CompilerConfiguration\">\n    <annotationProcessing>\n      <profile default=\"true\" name=\"Default\" enabled=\"true\" />\n      <profile name=\"Maven default annotation processors profile\" enabled=\"true\">\n        <sourceOutputDir name=\"target/generated-sources/annotations\" />\n        <sourceTestOutputDir name=\"target/generated-test-sources/test-annotations\" />\n        <outputRelativeToContentRoot value=\"true\" />\n        <module name=\"demo\" />\n        <module name=\"restserver\" />\n      </profile>\n    </annotationProcessing>\n  </component>\n  <component name=\"JavacSettings\">\n    <option name=\"ADDITIONAL_OPTIONS_OVERRIDE\">\n      <module name=\"demo\" options=\"-parameters\" />\n      <module name=\"restserver\" options=\"-parameters\" />\n    </option>\n  </component>\n</project>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/compiler.xml b/.idea/compiler.xml
--- a/.idea/compiler.xml	(revision 29d98c7b934bd4cc69b77ee85b4b595717524bc4)
+++ b/.idea/compiler.xml	(date 1672657357307)
@@ -8,13 +8,18 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <module name="demo" />
+        <module name="rest-server" />
         <module name="restserver" />
       </profile>
     </annotationProcessing>
+    <bytecodeTargetLevel>
+      <module name="rest-server" target="19" />
+    </bytecodeTargetLevel>
   </component>
   <component name="JavacSettings">
     <option name="ADDITIONAL_OPTIONS_OVERRIDE">
       <module name="demo" options="-parameters" />
+      <module name="rest-server" options="-parameters" />
       <module name="restserver" options="-parameters" />
     </option>
   </component>
Index: Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Dockerfile b/Dockerfile
new file mode 100644
--- /dev/null	(date 1672657340722)
+++ b/Dockerfile	(date 1672657340722)
@@ -0,0 +1,4 @@
+FROM openjdk:17
+COPY target/demo-0.0.1-SNAPSHOT.jar /tmp
+EXPOSE 8080
+ENTRYPOINT ["java", "-jar", "/tmp/demo-0.0.1-SNAPSHOT.jar"]
\ No newline at end of file
Index: target/classes/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>server.port = 8008\nspring.datasource.url=jdbc:mysql://localhost:3306/marketplace_harvest\nspring.datasource.username=root\nspring.datasource.password=password\nspring.jpa.database=mysql
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/target/classes/application.properties b/target/classes/application.properties
--- a/target/classes/application.properties	(revision 29d98c7b934bd4cc69b77ee85b4b595717524bc4)
+++ b/target/classes/application.properties	(date 1672657349088)
@@ -1,5 +1,5 @@
 server.port = 8008
-spring.datasource.url=jdbc:mysql://localhost:3306/marketplace_harvest
+spring.datasource.url=jdbc:mysql://mysqlcontainer:3306/marketplace_harvest
 spring.datasource.username=root
-spring.datasource.password=password
+spring.datasource.password=appdetex123
 spring.jpa.database=mysql
\ No newline at end of file
